mbedtls/library/AMBuilder

```diff
@@ -11,6 +11,8 @@ for cxx in builder.targets:
     os.path.join(builder.sourcePath, 'mbedtls', 'library'),
   ]
 
+  binary.compiler.cflags += ['-std=c11'];
```

Comment:
MSVC does not support `-std`:
> cl : Command line warning D9002 : ignoring unknown option '-std=c11'

And Python does not use `;` to end statements. So it should be:
```py
  if binary.compiler.family == 'gcc' or binary.compiler.family == 'clang':
    binary.compiler.cflags += ['-std=c11']
```